docker/depends/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------
+# PECAN FOR MODEL BASE IMAGE
+# ----------------------------------------------------------------------
+FROM rocker/tidyverse:3.5
```

Comment:
Could we also enable finer control of this with a variable? E.g.
```
R_VERSION=${R_VERSION:-3.5}
```

Reply:
Changed this, however to go to 3.6 we need to change it in 2 places (docker.sh and Dockerfile).